### DISCUSSION TITLE
Button has Disappeared Again

### DISCUSSION THREAD

**Discussion by MereMoon** [bijij/ViewImage#256] [Q&A] [answered]
Hello, I just wanted to give another update as the button has once again disappeared using Firefox on 06/14/2022. The go around seems to also not be working as well.

**Comment by davidtderrick** [comment]
Are we talking about the View Image extension? Yes, this has disappeared – again.

**Reply by MereMoon** [reply]
That is correct. Both the main button and the go-around are no longer working. At the moment, the best way to bypass the whole issue is the 'open image in new tab' action. Still gives you access to the full view but very frustrating that something is going on with Firefox and/or Google again.

**Comment by bijij** [comment, marked as answer]
You're correct google has seemingly once again made changes which break the extension, not sure if it's intentional, but I find it suspicious that the changes were made basically as soon as the last update was approved for chrome. I've pushed out an update which should start rolling out soon.
